.github/workflows/ci.yaml

```diff
@@ -259,20 +259,35 @@ jobs:
           java-version: ${{ env.INT_TEST_JAVA_RUNTIME_VERSION }}
           distribution: ${{ env.INT_TEST_JAVA_RUNTIME_VENDOR }}
 
+      - name: Start Azure Cosmos DB emulator
+        run: |
+          Write-Host "Launching Cosmos DB Emulator"
+          Import-Module "$env:ProgramFiles\Azure Cosmos DB Emulator\PSModules\Microsoft.Azure.CosmosDB.Emulator"
+          Start-CosmosDbEmulator -Consistency Strong
+
+      - name: Install TLS/SSL certificate
+        run: |
+          $cert = Get-ChildItem Cert:\LocalMachine\My | where{$_.FriendlyName -eq 'DocumentDbEmulatorCertificate'}
+          $params = @{
+            Cert = $cert
+            Type = "CERT"
+            FilePath = "$home/tmp-cert.cer"
+            NoClobber = $true
+          }
+          Export-Certificate @params
+          certutil -encode $home/tmp-cert.cer $home/cosmosdbcert.cer
+          Remove-Item $home/tmp-cert.cer
+          & ${env:JAVA_HOME}/bin/keytool.exe -keystore ${env:JAVA_HOME}/jre/lib/security/cacerts -storepass 'changeit' -importcert -noprompt -alias cosmos_emulator -file $home/cosmosdbcert.cer
+          & ${env:JAVA_HOME}/bin/keytool.exe -keystore ${env:JAVA_HOME}/jre/lib/security/cacerts -storepass 'changeit' -list -alias cosmos_emulator
+
       - name: Generate unique prefix using the epoch
         run: |
           echo "db_prefix=$(date +%s%3N)" >> $GITHUB_ENV
```

Comment:
I think we can remove this since only multiple integration test runners are not sharing the same CosmosDB instance now; there won't be database name conflicts without a prefix. 
I guess we could also remove the handling of the `scalardb.cosmos.database_prefix` environment variable in the integration test.

Reply:
@Torch3333 Good catch! Thank you. I'll remove `db_prefix` related jobs and parameters from the workflow file.

For Java code, I think we may want to test with real Cosmo DB later. The implementation handling `scalardb.cosmos.database_prefix` might be useful for such a case. I don't have a strong opinion on it, though. What do you think?

Reply:
Thanks.
Ok, keeping the implementation handling is fine.

Reply:
@komamitsu @Torch3333 I think we can remove `scalardb.cosmos.database_prefix`. This parameter is useful only when using a shared Cosmo DB account. If we want to test with real Cosmo DB later, we can create a new dedicated Cosmo DB account.

Reply:
@brfrn169 Thanks. Understood. I'll create a follow up PR for that.

Reply:
@komamitsu Thank you!